packages/database/src/migrations/20210628030723-AddNumberTeachersByEducationLevel-modifies-data.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, findSingleRecordBySql } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const CODE = 'LESMIS_teachers_education_level_district';
+
+const REPORT_CONFIG = {
+  fetch: {
+    dataElements: [
+      'noteach_pe_public_f',
+      'noteach_pe_public_m',
+      'noteach_pe_private_f',
+      'noteach_pe_private_m',
+      'noteach_se_public_f',
+      'noteach_se_public_m',
+      'noteach_se_private_f',
+      'noteach_se_private_m',
+    ],
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+        config: {
+          dataSourceEntityType: 'sub_district',
+        },
+      },
+    ],
+  },
+  transform: [
+    {
+      transform: 'select',
+      "'name'":
+        "translate($row.dataElement, { noteach_pe_public_f: 'Primary Education (Public)', noteach_pe_public_m: 'Primary Education (Public)', noteach_pe_private_f: 'Primary Education (Private)', noteach_pe_private_m: 'Primary Education (Private)', noteach_se_public_f: 'Secondary Education (Public)', noteach_se_public_m: 'Secondary Education (Public)', noteach_se_private_f: 'Secondary Education (Private)', noteach_se_private_m: 'Secondary Education (Private)' })",
+      '...': '*',
+    },
+    'keyValueByDataElementName',
+    {
+      transform: 'aggregate',
+      name: 'group',
+      '...': 'last',
+    },
+    {
+      transform: 'select',
+      "'Male'":
+        'sum([$row.noteach_pe_public_m, $row.noteach_pe_private_m, $row.noteach_se_public_m, $row.noteach_se_private_m])',
+      "'Female'":
+        'sum([$row.noteach_pe_public_f, $row.noteach_pe_private_f, $row.noteach_se_public_f, $row.noteach_se_private_f])',
+      '...': ['name'],
+    },
+    {
+      transform: 'select',
+      "'GPI'": '$row.Female/$row.Male',
+      '...': '*',
+    },
+    {
+      transform: 'select',
+      "'sort_order'":
+        "translate($row.name, { 'Primary Education (Public)': 1, 'Primary Education (Private)': 2, 'Secondary Education (Public)': 3, 'Secondary Education (Private)': 4 })",
+      '...': '*',
+    },
+    {
+      transform: 'sort',
+      by: '$row.sort_order',
+    },
```

Comment:
We would usually drop `sort_order` after this
```suggestion
    {
      transform: 'sort',
      by: '$row.sort_order',
    },
    {
      transform: 'select',
      '...': ['name', 'Male', 'Female', 'Total', 'GPI'],
    },
```